include/swift/AST/ClangModuleLoader.h

```diff
@@ -46,6 +48,16 @@ class ClangModuleLoader : public ModuleLoader {
   /// \returns true if there was an error adding the search path.
   virtual bool addSearchPath(StringRef newSearchPath, bool isFramework,
                              bool isSystem) = 0;
+
+  /// Determine whether \c overlayDC is within an overlay module for the
+  /// imported context enclosing \c importedDC.
+  ///
+  /// This routine is used for various hacks that are only permitted within
+  /// overlays of imported modules, e.g., Objective-C bridging conformances.
+  virtual bool isInOverlayModuleForImportedModule(
+                                            const DeclContext *overlayDC,
+                                            const DeclContext *importedDC) = 0;
```

Comment:
Indentation got messed up here (should be double-indented, not right-aligned).

Reply:
Double-intentation is awful

Reply:
Right-alignment is worse, since it's more likely to change if the function signature changes. But I'm not just applying my own style here; it's LLVM and clang-format style.